cypress/integration/views/custom-migration-target.view.ts

```diff
@@ -0,0 +1,12 @@
+export enum CustomMigrationTargetView {
+    createSubmitButton = "button[id='identity-form-submit']:contains('Create')",
+    editSubmitButton = "button[id='identity-form-submit']:contains('Save')",
+    actionsButton = "button[aria-label=Actions]",
+    editAction = "Edit",
+    deleteAction = "Delete",
+    nameInput = "input[name='name']:not(:hidden)",
+    descriptionInput = "input[name='description']:not(:hidden)",
+    imageInput = "div[name='imageID']:not(:hidden) > input[type='file']",
+    ruleInput = "div[name='customRulesFiles']:not(:hidden) > input[accept='.windup.xml']",
```

Comment:
If the locators are not standard for input, description and file , we can ask the UI team to standardize them .

we shouldn't have to use "not:hidden"

Reply:
None of the locators are standard right now because UI have two forms in the HTML at the same time
One for creating and another one for editing.

The problem is that both forms are in the HTML and they just hide and show a form depending on the action.

And both forms have the same selectors (even same ids) so this is the only way to identify the correct input as of now.

Reply:
@abrugaro we can start automation this way but let's ask Ian about this problem on slack and see if he can give us unique locators